File: z-clients/javascript/src/models/msgTopicConfigureIn.ts
// this file is @generated

export interface MsgTopicConfigureIn {
    namespace?: string | null;
    partitions: number;
}

export interface MsgTopicConfigureIn_ {
    namespace?: string | null;
    topic: string;
    partitions: number;
}

export const MsgTopicConfigureInSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): MsgTopicConfigureIn_ {
        return {
            namespace: object['namespace'],
            topic: object['topic'],
            partitions: object['partitions'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: MsgTopicConfigureIn_): any {
        return {
            'namespace': self.namespace,
            'topic': self.topic,
            'partitions': self.partitions,
        };
    }
}

File: z-clients/python/diom/models/msg_topic_configure_in.py
# this file is @generated

from ..internal.base_model import BaseModel


class MsgTopicConfigureIn(BaseModel):
    namespace: str | None = None

    partitions: int


class _MsgTopicConfigureIn(BaseModel):
    namespace: str | None = None

    topic: str

    partitions: int
